src/main/java/com/teksystems/TekcampExerciseSpring/Models/UserEntity.java

```diff
@@ -0,0 +1,139 @@
+package com.teksystems.TekcampExerciseSpring.Models;
+
+import java.util.Date;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.PrePersist;
+import javax.persistence.PreUpdate;
+import javax.persistence.Table;
+import javax.persistence.Transient;
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotNull;
+
+@Entity
+@Table(name="users")
+public class UserEntity {
+	@Id
+    @GeneratedValue(strategy=GenerationType.IDENTITY)
+    private Long id;
+	private String userId;
+	@NotNull(message = "First name cannot be null")
+	private String firstName;
+	@NotNull(message = "Last name cannot be null")
+	private String lastName;
+	@Email(message = "Email should be valid")
+    private String email;
+    private String password;
+    @Transient
```

Comment:
nice, but maybe if you don't want a field persisted, it shouldn't be in your entity at all? I can see why you'd want this field to confirm that a password is correct as intended, but that can be handled on the front end and this field is largely unnecessary on the back end, and definitely would only be a part of a request if anything